src/main/java/net/minecraftforge/common/config/IFieldWrapper.java

```diff
@@ -68,7 +68,7 @@
      * @param reqMCRestart True, if a change in this property requires a restart of Minecraft
      * @param reqWorldRestart True, if the world needs to be reloaded after changes to this property
      */
-    void setupConfiguration(Configuration cfg, String desc, String langKey, boolean reqMCRestart, boolean reqWorldRestart);
+    void setupConfiguration(Configuration cfg, String desc, String langKey, boolean reqMCRestart, boolean reqWorldRestart, boolean hasSlidingControl);
```

Comment:
Let's not break binary compat here, even if it is unlikely to affect anyone.
```suggestion
    default void setupConfiguration(Configuration cfg, String desc, String langKey, boolean reqMCRestart, boolean reqWorldRestart) {
        return setupConfiguration(cfg, desc, langKey, reqMCRestart, reqWorldRestart, false);
    }
    
    void setupConfiguration(Configuration cfg, String desc, String langKey, boolean reqMCRestart, boolean reqWorldRestart, boolean hasSlidingControl);
```